src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php

```diff
@@ -194,7 +194,16 @@ public static function register($mode = 0)
     private static function hasColorSupport()
     {
         if ('\\' === DIRECTORY_SEPARATOR) {
-            return false !== getenv('ANSICON') || 'ON' === getenv('ConEmuANSI') || 'xterm' === getenv('TERM');
+            return
+                defined('PHP_WINDOWS_VERSION_MAJOR')
```

Comment:
We just detected that the directory separator in `\`... I'm not 100% sure it's only used in Windows (there are more OSs out there than just *nix, Windows and Mac... What about OS2? Others?....)